src/plugins/intel_gpu/tests/test_cases/reverse_gpu_test.cpp

```diff
@@ -47,31 +47,70 @@ struct reverse_gpu_test : public ::testing::TestWithParam<ReverseParams<T, mode>
         auto data_type = type_to_data_type<T>::value;
         ReverseParams<T, mode> params = testing::TestWithParam<ReverseParams<T, mode>>::GetParam();
         auto& engine = get_test_engine();
-        tensor t;
 
-        auto reverse_input = engine.allocate_memory({data_type, params.input_format, params.input_tensor});
-        auto reverse_axes = engine.allocate_memory(
-            {ReverseModeTraits<mode>::data_type, format::bfyx, tensor(batch(1), feature(params.axis.size()))});
+        format fmt{format::bfyx};
+        bool reorder_needed = false;
+        switch (params.input_format) {
```

Comment:
I think it worth to extract this logic to separate function and maybe even move it to some common test utility

Reply:
like
out = getPlainFormatFor(fmt);
reorderNeeded = out != fmtl

Reply:
done